### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rohitxsh**

### CODE REVIEW

**Comment by andrewyatz** [comment]
Good progress. Before we merge in we need that systems diagram in place. Also for the evaluators I suggest bringing in a top-level paragraph which would:

- Link to the live UI deployment
- Describe how you query the interface (select species, select your filters and columns, mention the special logic for genomic queries, getting results)

File: README.md [hunk 1]
```diff
@@ -1,46 +1,180 @@
...
+<br>
+
+## UI snapshots:
+![Home page 1](./ui-snapshots/home_1.png)
+![Home page 2](./ui-snapshots/home_2.png)
+![Home page 3](./ui-snapshots/home_3.png)
+![Home page 4](./ui-snapshots/home_4.png)
+![Status page](./ui-snapshots/status.png)
+![Export page](./ui-snapshots/export.png)
+
```
Comment (marcoooo) [inline README.md:153]: May be update the screen shot with preview displayed.

**Review by marcoooo — APPROVED** [review]

**Review by bilalebi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rohitxsh) [reply README.md:153]: Donr

**Comment by rohitxsh** [comment]
@andrewyatz Updated the PR as per your review comments, please re-review

**Review by andrewyatz — APPROVED** [review]
Brilliant work. Looks perfect to commit from me.

**Pull request merged**